sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -1263,6 +1264,8 @@ object SQLContext {
    */
   @transient private val instantiatedContext = new AtomicReference[SQLContext]()
 
+  @transient private val sqlListener = new AtomicReference[SQLListener]()
```

Comment:
Just to make sure that it's not overlooked, please see my comment on the other PRs regarding whether this needs to actually hold a `SQLListener` instance or whether it can simply be an `AtomicBoolean`.